cloudify_cli/execution_events_fetcher.py

```diff
@@ -101,7 +131,10 @@ def wait_for_execution(client,
     events_fetcher = ExecutionEventsFetcher(client, execution.id,
                                             include_logs=include_logs)
 
-    # Poll for execution status until execution ends
+    # Poll for execution status and execution logs, until execution ends
+    # and we receive an event of type in WORKFLOW_END_TYPES
+    execution_ended = False
+    events_watcher = EventsWatcher(events_handler)
     while True:
```

Comment:
How do you feel placing "not(execution_ended and events_watcher.end_log_received)" as the condition instead of just True (since this block obviously has to end)?